common/protos/common/events.proto

```diff
@@ -62,6 +62,7 @@ message ContractTransaction {
   string func = 4;
   repeated bytes args = 5;
   uint64 replace_arg_index = 6;
+  repeated string participants = 7; 
```

Comment:
Can we call this `members` so we are consistent with terminology in the security domain and identity RFCs?

Reply:
okay